src/components/Pricing/PricingCalculator/index.tsx

```diff
@@ -99,6 +101,26 @@ export const PricingCalculator = () => {
                                 />
                             </div>
                         </div>
+                        <div className="border-b border-light dark:border-dark col-span-3 p-2 pl-10 relative">
+                            <span className="w-5 h-5 flex absolute top-3 left-3">{<FeatureFlags />}</span>
+                            <div className="flex flex-col lg:flex-row lg:justify-between lg:items-center">
+                                <strong>Feature flags</strong>
+                                <span>
+                                    <span className="text-lg font-bold">{featureFlagNumber.toLocaleString()}</span>{' '}
+                                    <span className="opacity-60 text-sm">requests</span>
+                                </span>
+                            </div>
+                            <div className="pt-4 pb-6">
+                                <LogSlider
+                                    stepsInRange={100}
+                                    marks={[1000000, 10000000, 100000000, 1000000000]}
+                                    min={1000000}
+                                    max={1000000000}
+                                    onChange={(value) => setFeatureFlagSliderValue(value)}
+                                    value={featureFlagSliderValue}
+                                />
+                            </div>
+                        </div>
```

Comment:
Feels like we can probably make this dynamic based on the available products and their tiers? That way we don't need to manually do this every time we add a new product.